hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/keyvalue/KeyValueHandler.java

```diff
@@ -827,12 +829,25 @@ ContainerCommandResponseProto handleGetSmallFile(
         ChunkBuffer data = chunkManager.readChunk(kvContainer, blockID,
             chunkInfo, dispatcherContext);
         dataBuffers.addAll(data.toByteStringList(byteBufferToByteString));
-        chunkInfoProto = chunk;
+        chunk.getChecksumData().getChecksumsList().stream().forEach(
+            (ck -> checksumBuilder.addChecksums(ck)));
```

Comment:
Please try to avoid streams on read/write path ([HDDS-3702](https://issues.apache.org/jira/browse/HDDS-3702)).

Reply:
Sure.